recipes/server.rb

```diff
@@ -63,7 +66,8 @@
   only_if { platform_family?('rhel') }
 end
 
-sysctl_param 'vm.swappiness' do
+sysctl_param 'set vm.swappiness to 0' do
+  key 'vm.swappiness'
```

Comment:
Just have the resource name be ``vm.swappiness``, then setting ``key`` is no longer required.

Reply:
@ramereth It has cloning conflict with base::sysctl since base::sysctl also has resource that set vm.swappiness. I've reverted it.